pipes/WDL/tasks/tasks_metagenomics.wdl

```diff
@@ -84,7 +85,7 @@ task krakenuniq {
 
   runtime {
     docker: "${docker}"
-    memory: "240 GB"
+    memory: select_first([machine_mem_gb, 240]) + " GB"
```

Comment:
this is a separate issue, but Simon's new krakenuniq database is now 282GB! Maybe we should bump this to 320GB or so (and explicitly ask dxWDL for a `mem3_ssd1_v2_x48`).

Reply:
Done.